demos/smart_classroom_demo/create_list.py

```diff
@@ -1,13 +1,10 @@
 #!/usr/bin/env python
 '''
  Copyright (C) 2018-2019 Intel Corporation
-
  Licensed under the Apache License, Version 2.0 (the "License");
  you may not use this file except in compliance with the License.
  You may obtain a copy of the License at
-
       http://www.apache.org/licenses/LICENSE-2.0
-
  Unless required by applicable law or agreed to in writing, software
  distributed under the License is distributed on an "AS IS" BASIS,
  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
```

Comment:
You shouldn't remove these lines, it is a standardized text of the license.

Reply:
Script from omz demo. I hit him when I was studying the demo.